devicon.json

```
@@ -6230,6 +6230,33 @@
         "color": "#00c48d",
         "aliases": []
     },
+    {
+        "name": "oauth",
+        "altnames": [],
+        "tags": [
+            "auth",
```

Not sure if we should use `auth` or `authentication`. 🤔
```suggestion
            "authentication",
```

---

`authentication` sounds more coherent & it also covers `auth` in it.